packages/components/psammead-radio-schedule/src/__snapshots__/index.test.jsx.snap

```diff
@@ -160,67 +150,57 @@ exports[`RadioSchedule should render ltr radio schedules correctly 1`] = `
 }
 
 @supports (display:grid) {
-  .emotion-41 {
+  .emotion-4 {
     display: block;
+    width: initial;
+    margin: 0;
   }
 
-@media (max-width:14.9375rem) {
-    .emotion-41 {
-      width: initial;
-      margin: 0;
```

Comment:
Emotion seems to be optimising this for us?  These two lines appear 6 times (once in every media query) in the "before"

In the after, they are just specified once, outside any media query

Accounts for 10 of the 40 removed lines